Dockerfile

```diff
@@ -56,71 +56,78 @@ COPY schema ./schema
 COPY scripts ./scripts
 COPY tools ./tools
 COPY vast ./vast
-COPY .clang-format .cmake-format LICENSE LICENSE.3rdparty README.md BANNER CHANGELOG.md CMakeLists.txt vast.yaml.example ./
+COPY LICENSE LICENSE.3rdparty README.md BANNER CHANGELOG.md CMakeLists.txt \
+    vast.yaml.example ./
 
 # Resolve repository-internal symlinks.
+# TODO: We should try to get rid of these long-term, as Docker does not work
+# well with repository-internal symlinks.
```

Comment:
Considering how much time we spent debugging this issue I think we should resolve it. I agree that it's annoying, but Docker has become a central part of our stack, so we should try to work with rather than against it.

Reply:
A line has to be drawn somewhere. If docker isn't able to integrate into our stack we should evaluate alternatives.

Reply:
That's a silly argument. All of these symlinks are replaceable with reasonable effort, but replacing the industry-standard container solution is not.

Please approve the PR and take the discussion elsewhere if this is your only concern. It is very much valid in the context of the Dockerfile at hand, but blocking this PR over it is most certainly not.